docs/rdiff-backup.1.md

```diff
@@ -745,6 +745,18 @@ special meaning in regex/glob patterns.
 
 All the available file selection conditions are listed under
 [SELECTION OPTIONS](#selection-options).
+
+Two principles need to be understood before really starting:
+
+1. pattern matching is stupid about paths, it just does pattern matching and
+   can't interpret patterns like path, especially it can't resolve absolute
+   into relative paths and vice-versa (compare with the '`-path`' option of
+   find).
+2. pattern matching is done on the complete path of each found file (no partial
+   matching and no file name matching).
+   Beware that complete path does _not_ mean full path, it can be a complete
+   relative path.
```

Comment:
It feels like a couple of examples here might help clarify things - e.g. "this pattern would match this path [...] but not this one [...]"?

Reply:
+1

Reply:
Good idea, done!